### ISSUE TITLE
`prose-p:text-justify` makes recipe less usable

### ISSUE THREAD

**Issue by jackwatters45** [withastro/docs#8357]
### 📚 Subject area/topic

Style rendered Markdown with Tailwind Typography recipe

### 📋 Page(s) affected (or suggested, for new content)

https://docs.astro.build/en/recipes/tailwind-rendered-markdown/

### 📋 Description of content that is out-of-date or incorrect

Tailwind class `prose-p:text-justify` is not wrong, but I would suggest it is better off omitted. I copied the recipe's prose component directly into my blog and did not notice the text-justification. It was quite annoying to debug and the recipe would result in a better experience if it was omitted.

```diff 
---
// src/components/Prose.astro
---
<div
  class="prose dark:prose-invert
  prose-h1:font-bold prose-h1:text-xl
  prose-a:text-blue-600 
- prose-p:text-justify
  prose-img:rounded-xl
  prose-headings:underline">
  <slot />
</div>
```

[Permalink](https://github.com/withastro/docs/blob/151f5baf3acdce25ac0c97802cd005653dd44235/src/content/docs/en/recipes/tailwind-rendered-markdown.mdx?plain=1#L70)

**Comment by at-the-vr** [comment]
Hey Jack 👋 I saw your Blog and adjusted the [letter spacing](https://tailwindcss.com/docs/letter-spacing#setting-the-letter-spacing) - here's the [commit](https://github.com/jackwatters45/portfolio-site/compare/main...at-the-vr:portfolio-site:main) && the [deployment](https://portfolio-site-mm7ujn6nq-atharvas-projects-7d110ede.vercel.app/blog/team-send) which looks much better to me.

clearly `text-justify` works well when paired with `letter-spacing`. I also tried the same recipe on Astro Blog template without `letter-spacing` and it went smooth as well. 

tldr: Gotta say no to removing the `prose-p:text-justify` because of an edge case. Hope you understand, thanks for testing the recipe a lot 🙌
